jpa/org.eclipse.persistence.jpa/src/main/java/org/eclipse/persistence/internal/jpa/StoredProcedureQueryImpl.java

```diff
@@ -527,6 +527,15 @@ public Object getOutputParameterValue(String parameterName) {
         return null;
     }
 
```

Comment:
/*
* Returns false if any query parameter is a named parameter, otherwise returns true.
*/

Reply:
hmm. I'm not sure I like this implementation. It feels sorta hackish, right? Checking all of the parameters to make a decision on one? Idk.